src/script/view_model/ContentViewModel.ts

```diff
@@ -217,17 +214,14 @@ export class ContentViewModel {
    *
    * @param conversation Conversation entity or conversation ID
    * @param options State to open conversation in
+   * @param domain TODO: Add description for domain param
    */
   readonly showConversation: ShowConversationOverload = async (
     conversation: Conversation | string,
     options: ShowConversationOptions,
     domain: string | null = null,
   ) => {
-    const {
-      exposeMessage: exposeMessageEntity,
-      openFirstSelfMention = false,
-      openNotificationSettings = false,
-    } = options;
+    const {openNotificationSettings = false} = options;
```

Comment:
Can you confirm that you restored those functionalities? (opening on the notification settings and scrolling to the first self mention)?